[AY1920S1-CS2113-T13-4] Duke Manager

maxxyx96 opened the pull request:
@talesrune
@e0318465
@dou-maokang
@gervaiseang
@maxxyx96

gervaiseang commented:
Implemented B-Reminders v1.0 but still left minor edits.

RubaP commented on src/main/java/duke/AddWindow.java:
```diff
@@ -0,0 +1,108 @@
+        duke = d;
+        this.mainWindow = mainWindow;
+        cbType.getItems().addAll(
+                "Todo",
```
Define these magical strings as constants (variable or enum or whatever the effective way)

RubaP commented on src/main/java/duke/AddWindow.java:
```diff
@@ -0,0 +1,108 @@
+    }
+
+    @FXML
+    void onMouseClickAdd() {
```
Again please handle all the magical strings in this method as constants

RubaP commented on src/main/java/duke/AddWindow.java:
```diff
@@ -0,0 +1,108 @@
+        System.out.println("clicked on " + cbType.getSelectionModel().getSelectedItem());
+
+        if (cbType.getSelectionModel().getSelectedItem().equals("Todo")) {
+            mainWindow.handleUserEvent("todo " + tfDesc.getText().trim());
```
Why does onMouseClickAdd method deal with string manipulations?

RubaP commented on src/main/java/duke/AddWindow.java:
```diff
@@ -0,0 +1,108 @@
+    }
+
+    @FXML
+    void onMouseClickReset() {
```
A good example for following SLAP

RubaP commented on src/main/java/duke/DialogBox.java:
```diff
@@ -0,0 +1,69 @@
+    }
+
+    /**
+     * Hi.
```
Why is there a 'Hi.' here?

RubaP commented on src/main/java/duke/Duke.java:
```diff
@@ -0,0 +1,215 @@
+     * @param filePathForContacts The location of the contact text file.
+     */
+    public Duke(String filePath1, String filePath2, String filePathForBudget, String filePathForContacts) {
+        ui = new Ui();
```
This violates SLAP and the readability is very poor since all the important objects creations and error handling happens here. Regarding SLAP, this method does ui.methodCall() and very lower level operation of creating objects as well.

RubaP commented on src/main/java/duke/Duke.java:
```diff
@@ -0,0 +1,215 @@
+                    budgetStorage.write(budgetList);
+                    cmd.executeStorage(items, ui, storage);
+                    break;
+                } else if (cmd instanceof ListPriorityCommand
```
Should the run method of Duke which is your starting point understand different low-level commands and react to it. I guess the responsibility fo run is to initiate the run flow by initiating the relevant objects. SRP is violated here

RubaP commented on src/main/java/duke/HelpWindow.java:
```diff
@@ -0,0 +1,80 @@
+        duke = d;
+        this.mainWindow = mainWindow;
+        cbCommands.getItems().addAll(
+                "todo",
```
Again the magical strings

RubaP commented on src/main/java/duke/HelpWindow.java:
```diff
@@ -0,0 +1,80 @@
+        EventHandler<ActionEvent> event = new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent actionEvent) {
+                if (cbCommands.getSelectionModel().getSelectedItem().equals("todo")) {
```
How about a switch case which will be more clear for the reader?

RubaP commented on src/main/java/duke/MainWindow.java:
```diff
@@ -0,0 +1,392 @@
+    private static final int ZERO = 0;
+    private static final int ONE = 1;
+    private static final int TIMER_DELAY = 500;
+    private static final int VBOX_WIDTH = 200;
```
Good

RubaP commented on src/main/java/duke/MainWindow.java:
```diff
@@ -0,0 +1,392 @@
+
+    @FXML
+    protected void handleUserEvent(String input) {
+        //String input = userInput.getText();
```
Please remove commented code

RubaP commented on src/main/java/duke/MainWindow.java:
```diff
@@ -0,0 +1,392 @@
+
+    @FXML
+    private void onMouseClick_ListView() {
+        //System.out.println("clicked on " + listT.getSelectionModel().getSelectedItem());
```
Again, please remove all the commented code

RubaP commented on src/main/java/duke/MainWindow.java:
```diff
@@ -0,0 +1,392 @@
+            dialogContainer.getChildren().add(
+                    DialogBox.getDukeDialog(response, dukeImage)
+            );
+            timer.schedule(exitDuke, new Date(System.currentTimeMillis() + TIMER_DELAY));
```
Again you are violating SLAP. If the responsibility of this method is to call relevant object's functionality to exist the program then it should not do lower-level string manipulation tasks to follow SLAP

RubaP commented on src/main/java/duke/command/AddBudgetCommand.java:
```diff
@@ -0,0 +1,69 @@
+    public void execute(TaskList items, Ui ui) {
+        ui.showAddBudget(amount, budgetList.getBudget());
+        budgetList.addToBudget(amount);
+        ui.showBudget(budgetList.getBudget());
```
Good example of following SLAP

RubaP commented on src/main/java/duke/command/HelpCommand.java:
```diff
@@ -0,0 +1,54 @@
+     */
+    @Override
+    public String executeGui(TaskList items, Ui ui) {
+        String str = Ui.helpRequest(items); //replace with events later on
```
Comments are for the reader to explain the code, not for the developer to remind the leftover jobs

RubaP commented on src/main/java/duke/command/UpdateCommand.java:
```diff
@@ -0,0 +1,200 @@
+            } else if (typeOfUpdate == 3) {
+                Task newtaskObj = null;
+                if (typeDesc.equals("todo")) {
+                    if (items.get(index) instanceof Todo) {
```
Three levels of 'if' statements are here. Try to simplify this method since it is very hard to follow now

RubaP commented on src/main/java/duke/command/UpdateCommand.java:
```diff
@@ -0,0 +1,200 @@
+                    } else {
+                        newtaskObj = new Todo(items.get(index).getDescription());
+                    }
+                } else if (typeDesc.equals("deadline")) {
```
Third level of 'if' can be changed to switch block to make it more clear and extracting out any reusable code snippet as method.

RubaP commented on src/main/java/duke/command/UpdateCommand.java:
```diff
@@ -0,0 +1,200 @@
+                        newtaskObj = new Todo(items.get(index).getDescription());
+                    }
+                } else if (typeDesc.equals("deadline")) {
+                    if (items.get(index) instanceof Event || items.get(index) instanceof Repeat) {
```
Same readability issue, if possible try to handle it in a more readable way (e.g. switch block, extract reusable code snippet as method)

RubaP commented on src/main/java/duke/parser/DateParser.java:
```diff
@@ -0,0 +1,44 @@
+                                            Integer.parseInt(mthInString),
+                                            Integer.parseInt(dayInString));
+        switch (datePeriod) {
+        case "days" : localDate = localDate.plusDays(1);
```
Please define all the magical strings as constants

RubaP commented on src/main/java/duke/storage/PriorityStorage.java:
```diff
@@ -0,0 +1,73 @@
+import java.util.ArrayList;
+
+public class PriorityStorage {
+    //protected String filePath = "./";
```
Remove commented code

RubaP commented on src/main/java/duke/storage/Storage.java:
```diff
@@ -0,0 +1,179 @@
+    //protected String filePath = "./";
+    protected String filePath = "";   //27-28, 40-47
+    String storageClassPath = Storage.class.getProtectionDomain().getCodeSource().getLocation().getPath();
+    private static final int ZERO = 0;
```
If these constants are repeatedly used by many classes, you can define it as an enum to increase the readability and maintainability

RubaP commented on src/main/java/duke/storage/Storage.java:
```diff
@@ -0,0 +1,179 @@
+                        items.add(t);
+                    }
+                } else if (commandList[ZERO].equals("D")) {
+                    if (taskDesc.trim().isEmpty() || dateDesc.trim().isEmpty()) {
```
So much of repeated code. Think of an effective way of reducing it by moving the repeated validation check to single place

RubaP commented on src/main/java/duke/task/ContactList.java:
```diff
@@ -0,0 +1,79 @@
+    }
+
+    /**
+     * To add a contact into the contact list.
```
The first name of the method should be in simple present e.g. Adds

RubaP reviewed (commented):
Overall, it is a good job. I havent repeatedly pointed out the same mistake. So everyone, please go through all the comments and address it if it is applicable to the code you wrote.

gervaiseang committed:
Merge remote-tracking branch 'origin/master'

talesrune committed:
Updated DetectAnomalies, updating aboutus, ui.png, and adding PPP.

talesrune committed:
Updated Ui.png

gervaiseang committed:
Merge pull request #8 from gervaiseang/branch-Reminder2

Reminders v2.1 done

talesrune committed:
Merge Conflicts resolved, removing dukeLogger.log (it is useless for now)

talesrune committed:
Fixed the javadoc issue

talesrune committed:
Merge pull request #168 from gervaiseang/master

Reminders v2.1 done

talesrune committed:
Fixed Merge Conflict, duke.txt issue.

talesrune committed:
Fixed javadoc on reminders

talesrune committed:
Merge pull request #169 from talesrune/master

Updated DetectAnomalies, updating aboutus, ui.png, and adding PPP.

gervaiseang committed:
Update UserGuide.adoc

talesrune committed:
Updated UserGuide.adoc

talesrune committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

talesrune committed:
Merge pull request #170 from talesrune/master

Updated Userdoc

talesrune committed:
Fixing some issues with the GUI.

talesrune committed:
Merge pull request #172 from talesrune/master

Fixing some issues with the GUI.

e0318465 committed:
Branching to new branch

maxxyx96 committed:
Implemented Undo budget command, Optimised some output texts for Ui.

maxxyx96 committed:
Merge pull request #173 from maxxyx96/master

Implemented Undo budget command, Optimised some output texts for Ui.

talesrune committed:
Added GUI for budget list.

talesrune committed:
removing logger

talesrune committed:
Merge pull request #206 from talesrune/master

Adding Budget List Window for GUI

talesrune committed:
Implemented Enums for Numbers 1 - 31.

talesrune committed:
Added ErrorMessages Enum (added some from parser)

talesrune committed:
Merge pull request #207 from talesrune/master

Implemented Enums

e0318465 committed:
Moved repeat function to unused

e0318465 committed:
Staging log file

e0318465 committed:
Removed repeat function and updated detectduplicate test case

e0318465 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

e0318465 committed:
Merge branch 'RemoveCode'

# Conflicts:
#	data/duke.txt
#	dukeLogger.log
#	src/main/java/duke/command/DuplicateFoundCommand.java
#	src/main/java/duke/command/FilterCommand.java
#	src/main/java/duke/command/UpdateCommand.java
#	src/main/java/duke/parser/Parser.java
#	src/main/java/duke/storage/Storage.java
#	src/main/java/duke/task/Repeat.java
#	src/main/java/duke/ui/Ui.java
#	src/test/java/duke/command/UpdateCommTest.java

talesrune committed:
Implementing SLAP on some java files.

talesrune committed:
Merge branch 'branch-SLAP'

talesrune committed:
Merge pull request #208 from talesrune/master

Implementing SLAP on some java files.

e0318465 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

e0318465 committed:
Improved test cases for detect duplicate and contacts

e0318465 committed:
Test case javadocs done

talesrune committed:
Updated filepath, works on Windows and Linux.

maxxyx96 committed:
Delete DeveloperGuide.adoc

maxxyx96 committed:
Delete UserGuide.adoc

User guide and Developer guide backup is with user maxxyx96, contact him if you require the latest change before deletion.

maxxyx96 committed:
Implemented budget to store maximum of 2 dp

maxxyx96 committed:
Merge branch 'budget2dp'

e0318465 committed:
Merge pull request #209 from maxxyx96/master

Values can store a maximum of 2dp, addresses Issue #178

e0318465 committed:
Updated code according to the issues raised

e0318465 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

e0318465 committed:
Merge branch 'ImplementCodeQuality'

e0318465 committed:
Cleaned up the code

maxxyx96 committed:
Delete dukeLogger.log

Not necessary to include.

maxxyx96 committed:
Merge pull request #210 from e0318465/master

Added test cases and improved on contacts code

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

talesrune committed:
Merge branch 'branch-FilePath'

gervaiseang committed:
Merge branch 'master' into branch-Help

talesrune committed:
Fixed javadoc

talesrune committed:
Merge pull request #211 from talesrune/master

Updated FilePath

maxxyx96 committed:
Moved GUI-related code to Ui, refactored some variables.

maxxyx96 committed:
Merge pull request #212 from maxxyx96/master

Moved GUI-related code to Ui, refactored some variables.

talesrune committed:
Removed CLI portion of the code.

talesrune committed:
Merge branch 'branch-RemoveCLI'

talesrune committed:
Merge pull request #213 from talesrune/master

CLI portion is removed

e0318465 committed:
Removed certain comments written wrongly

e0318465 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

# Conflicts:
#	dukeLogger.log
#	src/main/java/duke/command/AddBudgetCommand.java

Dou-Maokang committed:
update

talesrune committed:
Added Contactslist to GUI

Dou-Maokang committed:
Merge pull request #214 from Dou-Maokang/master

update

gervaiseang committed:
Help update 1

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

gervaiseang committed:
Help update 1

gervaiseang committed:
Merge branch 'branch-Help'

gervaiseang committed:
Merge pull request #215 from gervaiseang/master

Help update 1

e0318465 committed:
Added more data in contact.txt file

e0318465 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

talesrune committed:
Bug Fixes from github.

talesrune committed:
Merge Conflicts resolved:
#	src/main/java/duke/ui/HelpWindow.java

talesrune committed:
Merge pull request #217 from talesrune/master

Bug Fixes

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

gervaiseang committed:
Merge branch 'master' into branch-Help

gervaiseang committed:
Help update 2

gervaiseang committed:
Merge pull request #10 from gervaiseang/branch-Help

Help update 2

gervaiseang committed:
Merge pull request #218 from gervaiseang/master

Help v2.1

e0318465 committed:
Updated author

e0318465 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

e0318465 committed:
Fixed contactlist method

e0318465 committed:
Merge pull request #219 from e0318465/master

Updated contacts and unused codes

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

e0318465 committed:
Debugged addcontacts input

e0318465 committed:
Merge pull request #220 from e0318465/master

Debugged addcontacts input

maxxyx96 committed:
Update Reposense Link

Was not captured by bot so attempting to use another link.

maxxyx96 committed:
Optimised friendlierSyntax

maxxyx96 committed:
Merge branch 'FriendliersyntaxUpdate'

maxxyx96 committed:
Refactored parser's constant array length check.

maxxyx96 committed:
Optimised methods

maxxyx96 committed:
Merge pull request #221 from maxxyx96/master

Optimised and Refactored portion of parser

e0318465 committed:
Improve code quality - Made addcontact more readable in the code and avoided deep nesting in deletecontact

e0318465 committed:
Shortened code in deletecontact and made a better description in javadoc

e0318465 committed:
Made the happy path more prominent for deletecontactcommand

maxxyx96 committed:
Demo auto writes in launcher, not in jar yet.

e0318465 committed:
Realised the email checker was omitted after code changes. Added it back

maxxyx96 committed:
Implemented auto creation of sample/demo data if no files are detected.

maxxyx96 committed:
Merge pull request #222 from maxxyx96/master

Read in sample code from jar

e0318465 committed:
Updated test case for contact

e0318465 committed:
Merge branch 'Debugs'

e0318465 committed:
Merge pull request #223 from e0318465/master

Implemented code quality for contacts

talesrune committed:
Updated sample list code.

talesrune committed:
Merge branch 'branch-Sample'

Conflicts resolved.

# Conflicts:
#	data/duke.txt
#	src/test/java/duke/command/ContactsCommTest.java

talesrune committed:
Merge pull request #224 from talesrune/master

Updated sample file path

e0318465 committed:
Changed the exception test for contactscommtest to the correct way

e0318465 committed:
Added one more test case for listcontact

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

e0318465 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

# Conflicts:
#	src/test/java/duke/command/ContactsCommTest.java

e0318465 committed:
Merge pull request #225 from e0318465/master

Updated exception testing with the correct way

gervaiseang committed:
Merge remote-tracking branch 'upstream/master' into branch-Reminder2

gervaiseang committed:
delete unused imports

gervaiseang committed:
To remove unused code

gervaiseang committed:
Merge pull request #11 from gervaiseang/branch-Reminder2

To remove unused code

e0318465 committed:
Moved unused Ui code to the unused folder

e0318465 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

e0318465 committed:
Moved unused Ui code to the unused folder (small fix)

e0318465 committed:
Merge pull request #227 from e0318465/master

Moved unused Ui codes to unused

maxxyx96 committed:
Duke will now autosave when pressing the "X" button at the top right corner

maxxyx96 committed:
Merge pull request #228 from maxxyx96/master

Duke will now autosave when pressing the "X" button

talesrune committed:
Fixed up Priority list and find date.

talesrune committed:
Fixed more bugs.

talesrune committed:
Merge branch 'branch-FixPriority'

talesrune committed:
Merge pull request #229 from talesrune/master

Fixed Priority commands and finddate

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

gervaiseang committed:
Merge pull request #226 from gervaiseang/master

To remove unused code

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

gervaiseang committed:
Merge branch 'master' into branch-Reminder2

gervaiseang committed:
Move unused code into unused folder

gervaiseang committed:
Fixed checkstyle

gervaiseang committed:
Merge pull request #13 from gervaiseang/branch-Reminder2

Move unused code into unused folder

gervaiseang committed:
Merge remote-tracking branch 'origin/master'

maxxyx96 committed:
Added acknowledgements from references

Dou-Maokang committed:
eliminated umbiguity in fixeddate

maxxyx96 committed:
Implemented closing of handlers on DukeLogger

maxxyx96 committed:
Merge pull request #231 from gervaiseang/master

Move unused code into unused folder and test Codacy integration

maxxyx96 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main into temp

maxxyx96 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

maxxyx96 committed:
Merge branch 'temp'

Dou-Maokang committed:
updated priority

Dou-Maokang committed:
updated Parser for finddate

Dou-Maokang committed:
added author

talesrune committed:
Added Credits Gui.

Dou-Maokang committed:
updated Parser

talesrune committed:
Merge branch 'branch-Credits'

talesrune committed:
Merge branch 'master' into master

talesrune committed:
Merge pull request #233 from Dou-Maokang/master

fixed finddate & fixed duration

maxxyx96 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

talesrune committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

talesrune committed:
Removed remind from parser again....

talesrune committed:
Merge pull request #235 from talesrune/master

Added Credits GUI

maxxyx96 committed:
Updated acknowledgements

maxxyx96 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

maxxyx96 committed:
Handled closing of logger, Acknowledged sources.  (#236)

* Added acknowledgements from references

* Implemented closing of handlers on DukeLogger

* Updated acknowledgements

maxxyx96 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

gervaiseang committed:
Merge branch 'master' into branch-Help

Dou-Maokang committed:
fixed priority list and item list size mismatch problem

gervaiseang committed:
help command will pop up HelpWindow

maxxyx96 committed:
Optimised saving on forced exit such that it wont work by task manager's end task.

maxxyx96 committed:
Merge pull request #238 from maxxyx96/master

Optimised saving of state for forced exit

Dou-Maokang committed:
fixed javastyle

Dou-Maokang committed:
Merge pull request #239 from Dou-Maokang/master

fixed priority list and item list size mismatch problem

gervaiseang committed:
Help finalVersion done

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

gervaiseang committed:
fixed checkstyle

gervaiseang committed:
Merge pull request #14 from gervaiseang/branch-Help

Help finalVersion done

gervaiseang committed:
fixed Codacy

gervaiseang committed:
Merge pull request #240 from gervaiseang/master

Help finalVersion done

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

maxxyx96 committed:
updated sample resources

maxxyx96 committed:
Refactored code according to codacy

maxxyx96 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

maxxyx96 committed:
Reverse files detected to be more issue prone in codacy

maxxyx96 committed:
Merge pull request #242 from maxxyx96/master

Added sample resources, improved some code

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

e0318465 committed:
Removed redundant statement in CONTACT_FORMAT

e0318465 committed:
Merge pull request #243 from e0318465/master

Removed redundant statement in CONTACT_FORMAT

maxxyx96 committed:
Added help for friendlier syntax & credits

maxxyx96 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

maxxyx96 committed:
Added backup information in help

maxxyx96 committed:
Merge pull request #244 from maxxyx96/master

Added help info & credits

talesrune committed:
Fixed the sample priority data

talesrune committed:
Date Checking improved.

talesrune committed:
Merge Conflicts resolved.

# Conflicts:
#	data/priority.txt
#	src/main/resources/sample/priority.txt

maxxyx96 committed:
code formatting

maxxyx96 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

maxxyx96 committed:
Merge pull request #246 from maxxyx96/master

code formatting

talesrune committed:
Moving System.exit(Numbers.ZERO.value); somewhere else

talesrune committed:
Stupid Codacy

talesrune committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

talesrune committed:
Merge pull request #248 from talesrune/master

Improved date checking

talesrune committed:
Fixed Date parsing, and the system.exit

talesrune committed:
Merge pull request #249 from talesrune/master

Fixed Date parsing, and the system.exit

gervaiseang committed:
Merge remote-tracking branch 'upstream/master'

gervaiseang committed:
Added test cases

gervaiseang committed:
Merge branch 'master' into branch-Help

gervaiseang committed:
Merge pull request #15 from gervaiseang/branch-JunitTest

Added test cases

gervaiseang committed:
Merge remote-tracking branch 'origin/master'

gervaiseang committed:
Merge branch 'master' into branch-Help

gervaiseang committed:
Added priority

gervaiseang committed:
Merge pull request #16 from gervaiseang/branch-Help

Added priority

gervaiseang committed:
Merge pull request #250 from gervaiseang/master

Added test cases and add priority command

e0318465 committed:
Final update for detectduplicate

e0318465 committed:
Merge branch 'FinalChanges'

e0318465 committed:
Merge pull request #251 from e0318465/master

Final change for detectduplicate

e0318465 committed:
Small edit

e0318465 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

e0318465 committed:
Merge pull request #252 from e0318465/master

Small edit

maxxyx96 committed:
Attempting coveralls

maxxyx96 committed:
Update Ui.png

maxxyx96 committed:
Removed Coveralls attempt, Updated Ui Png

maxxyx96 committed:
Updated Ui.png (#253)

* Updated ui.png

maxxyx96 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113-T13-4/main

talesrune committed:
QuickFix for finddate

maxxyx96 committed:
update friendlier syntaxs

talesrune committed:
Codacy

maxxyx96 committed:
Updated help for friendliersyntax command

maxxyx96 committed:
Merge pull request #256 from maxxyx96/master

update friendliersyntax

talesrune committed:
Fixing up finddate code

talesrune committed:
Merge pull request #257 from talesrune/master

 QuickFix for finddate

maxxyx96 committed:
Update Responsibilities and Roles

e0318465 committed:
Fixed Alignment

maxxyx96 committed:
Update AboutUs.adoc

talesrune committed:
Major cleaning!!!

talesrune committed:
Merge pull request #258 from talesrune/master

Major cleaning!!!

talesrune committed:
cleaning codes

talesrune committed:
Merge branch 'branch-cleaning2'

talesrune committed:
Merge pull request #259 from talesrune/master

Cleaning code #2

talesrune committed:
Sample code updated.

talesrune committed:
Updated Docs

talesrune committed:
Merge branch 'branch-Doc'

talesrune committed:
Merge pull request #260 from talesrune/master

Updated PPP

e0318465 committed:
Final PPP

e0318465 committed:
Outdated

maxxyx96 committed:
Upload UG and DG in pdf format

maxxyx96 committed:
Update DG

maxxyx96 committed:
Update PPP for maxxyx96

maxxyx96 committed:
Rename Guides

maxxyx96 committed:
Update AboutUs.adoc

gervaiseang committed:
Uploaded latest PPP

e0318465 committed:
Updated link

maxxyx96 committed:
Merge pull request #261 from maxxyx96/master

Update UG, DG, PPP

gervaiseang committed:
Latest PPP

gervaiseang committed:
Delete [AY1920S1-CS2113-T13-4][Gervaise Ang]PPP.pdf

gervaiseang committed:
Delete gervaiseang.adoc

gervaiseang committed:
Latest PPP

gervaiseang committed:
Delete gervaiseang.pdf.pdf

maxxyx96 committed:
Fixed ug dg link

gervaiseang committed:
Update README.adoc
